wls-exporter-core/src/main/java/com/oracle/wls/exporter/NotFoundException.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package com.oracle.wls.exporter;
+
+/**
+ * An exception indicating that the client could not connect to the specified URL.
+ */
+public class NotFoundException extends WebClientException {
```

Comment:
I can't seem to find any usages of this new class

Reply:
Nice catch, thanks.